### ISSUE TITLE
Screen reader issues

### ISSUE THREAD

**Issue by cyrmax** [farmerbb/Taskbar#340]
Hello!
The app is awesome but the tray itself lacks accessibility for screen readers for blind users.

* Every app icon in tray is announced as "Entry" instead of saying app name.
* Some buttons in tray have no readable accessibility labels and are announced as "button".
* The main switch on the app's main screen is announced like "On switch" without descriptive accessibility label.
* The main app settings screen contains some strange not clickable element which is announced as "new line" (probably really just a new line symbol in accessibility label or "new line" words).

If these accessibility issues are fixed, the whole blind community will be very happy :) 

Here is a video demonstration of all mentioned problems.

Excuse please that non-English text announced by my screen reader sometimes; this is in my native language. But these just emojies and numbers, nothing more.

https://drive.google.com/file/d/1y4-73z-YDl2YUP840xbFlHH2-OL9tUEX/view?usp=sharing
